### ISSUE TITLE
My bot won't stop

### ISSUE THREAD

**Issue by IDH258** [steamship-packages/autogpt-telegram-chatbot#5]
I asked for a few copywriting sentences and it just started to randomly select other tasks to do and now won't stop, just keeps on going. It started to give me a to do list on how to deep clean my bathroom 😂 and much more random stuff.

**Comment by vertalius** [comment]
Yep, my bot babbled random tasks about two days from the very first message  😒 And yes - that bot of mine also ended his 48 hour monologue with some kind of tutorial how to clean my home 🤣
